Cargo.lock

```diff
@@ -195,7 +195,7 @@ source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "0d0864d84b8e07b145449be9a8537db86bf9de5ce03b913214694643b4743502"
 dependencies = [
  "quote 1.0.6",
- "syn 1.0.17",
+ "syn 1.0.33",
```

Comment:
Why was this updated without any `Cargo.toml` change?

Reply:
That's due to adding `serde` in sc-cli.